Mathlib/Geometry/Euclidean/Angle/Unoriented/TriangleInequality.lean

```diff
@@ -0,0 +1,223 @@
+/-
+Copyright (c) 2025 Ilmārs Cīrulis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ilmārs Cīrulis
+-/
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Basic
+
+/-!
+# The Triangle Inequality for Angles
+
+This file contains proof that angles obey the triangle inequality.
+-/
+
+open InnerProductGeometry
+
+variable {V : Type}
+variable [NormedAddCommGroup V]
+variable [InnerProductSpace ℝ V]
+
+/-- **Gets unit length vector from given vector** -/
+noncomputable def unit (x : V): V := ‖x‖⁻¹ • x
```

Comment:
We use this construction several times in Mathlib, see https://loogle.lean-lang.org/?q=%E2%80%96%3Fa%E2%80%96%E2%81%BB%C2%B9+%E2%80%A2+%3Fa
Possibly, it deserves a `def` but I think that the name `unit` is too generic.

Reply:
As well as a less generic name, this definition should go in a general normed space file. Likewise, any other lemmas that are generic to normed spaces should go in such a file; lemmas involving the inner product but not connected to angles should probably go in a general inner product space file.